pkg/config/application_test.go

```diff
@@ -702,3 +702,33 @@ func TestCustomSyncConfig(t *testing.T) {
 		})
 	}
 }
+
+// TODO: Add testcases for other kinds of applications.
```

Comment:
What do we want here? I don't see any kinds here 👀

Reply:
[b1e6555](https://github.com/pipe-cd/pipecd/pull/4720/commits/b1e65550b7468e808996ec625c578c076281c6a6)
Sorry this comment is meaningless 🙏  so deleted